exporter/awsemfexporter/datapoint.go

```diff
@@ -371,18 +371,13 @@ func (dps summaryDataPointSlice) IsStaleNaNInf(i int) (bool, pcommon.Map) {
 
 // createLabels converts OTel AttributesMap attributes to a map
 // and optionally adds in the OTel instrumentation library name
-func createLabels(attributes pcommon.Map, instrLibName string) map[string]string {
+func createLabels(attributes pcommon.Map) map[string]string {
 	labels := make(map[string]string, attributes.Len()+1)
 	attributes.Range(func(k string, v pcommon.Value) bool {
 		labels[k] = v.AsString()
 		return true
 	})
 
-	// Add OTel instrumentation lib name as an additional label if it is defined
-	if instrLibName != "" {
-		labels[oTellibDimensionKey] = instrLibName
-	}
-
```

Comment:
This is removing labels from produced metrics, no?  Isn't that a significantly breaking change?

Reply:
Unless it's referenced in [metric_declaration](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awsemfexporter/README.md#metric_declaration), it won't impact produced metrics.

Reply:
It was discussed that the original intention was to reduce payload size, but it will still pose a breaking change regardless. This functionality has been restored in latest commit.